components/Sidebar/panels/index.tsx

```diff
@@ -1,12 +1,20 @@
 import Friends from "./Friends";
 import Notifications from "./Notifications";
+import Playback from "./Playback/Playback";
```

Comment:
Same as above, this should import an index.tsx file inside of playback (and that's where you could put isTrackPlaying and URI)

```suggestion
import Playback from "./Playback";
```